lm_eval/api/task.py

```diff
@@ -1002,37 +1002,38 @@ def process_results(self, doc, results):
                 choices = self.doc_to_choice(doc)
                 gold = choices[gold]
 
-            for key, result in zip(self._metric_fn_list.keys(), results):
-                if self.multiple_target:
-                    # in the case where we have multiple targets,
-                    # return true if any are true
-                    # TODO: this may break for multipLe_target, non zero-or-1 metrics
-                    scores = []
-                    for gold_option in gold:
-                        res = self._metric_fn_list[key](
-                            references=[gold_option],
+            for metric in self._metric_fn_list.keys():
+                for result in results:
```

Comment:
in what circumstance do we expect `results` to be a list with more than one elem?

Reply:
Hmmm, you're right, it should actually just be one. 

`results = results[0]`

Do you think we should include loglikelihood score for greedy until? The likelihood of a generation that is.

Reply:
We don't currently measure loglikelihood in greedy_until and couldn't on API models. if there's a clean way to (optionally) grab loglikelihood of the generation from a HF model as we go then that could be nice, but I'm not aware of one? Do you know?

Reply:
Not sure I do, we can pin this for another task. It would be useful for tasks like squadv2 we talked about the other day.